### PR TITLE
Update .NET Core get started articles (console app and class library)

### PR DESCRIPTION
**Pull request by gewarren**
Combine C# and VB articles for first console app and class library. Also update them for VS 2019.

Contributes to #16046

### CODE REVIEW

**Comment by gewarren** [comment]
Close/reopen to trigger build.

**Comment by Thraka** [comment]
How come there are tabs? Because of the different ways of creating a project and adding the screenshots? :( I find this will be confusing for people. They click on VB because they want VB but they don't know to click on the VB language indicator at the top so they still see C# code.

**Comment by gewarren** [comment]
> How come there are tabs? Because of the different ways of creating a project and adding the screenshots? :( I find this will be confusing for people. They click on VB because they want VB but they don't know to click on the VB language indicator at the top so they still see C# code.

I agree it's confusing. I did add this tip below the code snippet - do you think it helps?

image

Honestly though, I think we could probably do away with the tabs. To me, the maintenance cost is too high to justify it. Also, I'm asking myself why we even have these topics. Shouldn't we let the VS team handle getting started tutorials like this?

**Comment by Thraka** [comment]
I guess it would help, but it's really ugly 😢 I think this is the original reason why we had the VB and C# topics. Our system doesn't let us do this properly.

**Comment by mairaw** [comment]
I think we should just use the tabs on the article and not mix and match tabs and selector.

**Comment by gewarren** [comment]
> I think we should just use the tabs on the article and not mix and match tabs and selector.

Okay, I'll do that, but if I had the choice I'd just use selectors. There's too much duplication of content with the tabs, because it seems like if you have a numbered list, the whole thing has to be duplicated. You can't just tab out a section of the numbered list. So when you go in and correct or change something, you have to remember to do it in multiple places.

> I think basic stuff, first time in "visual studio create new project" articles should be on visual studio, but anything deeper is us and we're using visual studio as a tool.

Currently I'm working on the debugging topic, and I don't see anything unique to .NET Core in this topic. It's really about how to get started using the debugger in Visual Studio. Should we really be covering that in the .NET docs?

**Comment by mairaw** [comment]
I don't think we need to cover that here.

**Comment by gewarren** [comment]
@BillWagner I added the VB tabs back and removed the language selectors. It's ready for review. Thanks!

File: docs/core/tutorials/with-visual-studio.md [hunk 1]
```diff
@@ -1,57 +1,82 @@
...
 
-1. On the **Create a new project** window, select the following filters:
+   1. On the start window, choose **Create a new project**.
 
-   - **C#** in the **All Languages** dropdown. 
-   - **Console** in the **All Project Types** dropdown.
+      ![Create a new project button selected on the Visual Studio start window](media/with-visual-studio/start-window.png)
 
-   After the filters are applied, choose the **Console App (.NET Core)** template and choose **Next**.
+   1. On the **Create a new project** page, enter **console** in the search box. Next, choose **C#** from the Language list, and then choose **Windows** from the Platform list. Choose the **Console App (.NET Core)** template, and then choose **Next**.
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio.md]: Same point on platform:

```suggestion
   1. On the **Create a new project** page, enter **console** in the search box. Next, choose **C#** from the Language list, and then choose **All platforms** from the Platform list. Choose the **Console App (.NET Core)** template, and then choose **Next**.
```

File: docs/core/tutorials/with-visual-studio.md [hunk 2]
```diff
@@ -1,57 +1,82 @@
...
+
+1. Open Visual Studio 2019.
+
+1. Create a new Visual Basic .NET Core console app project named "HelloWorld".
+
+   1. On the start window, choose **Create a new project**.
+
+      ![Create a new project button selected on the Visual Studio start window](media/with-visual-studio/start-window.png)
+
+   1. On the **Create a new project** page, enter **console** in the search box. Next, choose **Visual Basic** from the Language list, and then choose **Windows** from the Platform list. Choose the **Console App (.NET Core)** template, and then choose **Next**.
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio.md]:
```suggestion
   1. On the **Create a new project** page, enter **console** in the search box. Next, choose **Visual Basic** from the Language list, and then choose **All platforms** from the Platform list. Choose the **Console App (.NET Core)** template, and then choose **Next**.
```

**Review by BillWagner — APPROVED** [review]
This is great work @gewarren I like the way you've combined the VB and C# versions in a seamless experience.

I had a few suggestions relating to the selected platform. After that, you can :shipit:

**Comment by gewarren** [comment]
Thanks for the review Bill!

**Pull request merged**